client/app/components/ui/collapsible-text.js

```diff
@@ -0,0 +1,12 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+
+export default class CollapsibleTextComponent extends Component {
+  @tracked showAdditionalText = false;
```

Comment:
Are we able to do without `@tracked` ? It seems it's not necessary here, since there are no getters/computeds using `showAdditionalText`

Reply:
I think @tracked is necessary here because we're updating it in an action. It fails if it's not @tracked